packages/shared/src/components/Settings.tsx

```diff
@@ -116,7 +117,9 @@ export default function Settings({
         <CustomSwitch
           inputId="layout-switch"
           name="insaneMode"
-          leftContent={<CardIcon secondary={!insaneMode} />}
+          leftContent={
+            <CardIcon size={IconSize.XSmall} secondary={!insaneMode} />
```

Comment:
Was it intended to intorudce the `XSmall` to this one?

Reply:
Ah yes forgot to mention that, it was way to big originally (the non active one) so adjusted how we scale them in this new fix.